pkg/scheduler/scheduling_queue_test.go

```diff
@@ -50,6 +49,13 @@ func createRootQueue(maxRes map[string]string) (*SchedulingQueue, error) {
 	return newSchedulingQueueInfo(root, nil), err
 }
 
+// create the root queue with no maxRes and fail if it couldn't be created
+func createDefaultRootQueueProtected(t *testing.T) *SchedulingQueue {
+	queue, err := createRootQueue(nil)
+	assert.NilError(t, err, "queue create failed")
+	return queue
+}
+
```

Comment:
Same as for the info objects, don't hide the error in a function.
This reversal will also require variable declaration to be reversed to prevent shadowing